resources/resincli.robot

```diff
@@ -69,6 +69,11 @@ Device ${device_uuid} log should contain ${value}
     Process ${result}
     Should Contain    ${result.stdout}    ${value}
 
+Check if host OS version is same through resin cli
```

Comment:
This keyword assumes a global variable os_version and a device uuid. Please make it generic.
Example: Check if host OS version of device ${device_uuid} is ${os_version}